habitat/core/spaces.py

```diff
@@ -100,7 +106,7 @@ def contains(self, x):
         if not isinstance(x, Sized):
             return False
 
-        if self.min_seq_length <= len(x) <= self.max_seq_length:
+        if len(x) < self.min_seq_length or len(x) > self.max_seq_length:
```

Comment:
Maybe `if not( self.min_seq_length <= len(x) <= self.max_seq_length):` for shorter form.